google-cloud-storage/src/main/java/com/google/cloud/storage/UnifiedOpts.java

```diff
@@ -1399,6 +1407,14 @@ public Mapper<UpdateBucketRequest.Builder> updateBucket() {
     }
   }
 
+  static final class EnableObjectRetention extends RpcOptVal<Boolean> implements BucketTargetOpt {
+    private static final long serialVersionUID = -2581147719605551578L;
+
+    private EnableObjectRetention(boolean val) {
+      super(StorageRpc.Option.ENABLE_OBJECT_RETENTION, val);
+    }
```

Comment:
since this doesn't support grpc yet, can you define an erroring mapper for the updateBucket() method similar to what was done with match glob in https://github.com/googleapis/java-storage/pull/1965/files#diff-4162f80a70285d5bd410d76a86cf59b5fa513dc4ad8e093115926eddda94f68bR1085-R1089